test/faker/name/pt_br_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Faker.Name.PtBrTest do
+  use ExUnit.Case, async: true
+  
+  test "first_name/0" do
+    assert is_binary(Faker.Name.first_name)
+  end
+
+  test "last_name/0" do
+    assert is_binary(Faker.Name.last_name)
+  end
+
+  test "name/0" do
+    assert is_binary(Faker.Name.name)
+  end
+
+  test "prefix/0" do
+    assert is_binary(Faker.Name.prefix)
+  end
+
+  test "suffix/0" do
+    assert is_binary(Faker.Name.suffix)
+  end
+end
```

Comment:
Note: Your editor seems to not append New-Line to EOF automagically :) For some reason, you should always finish a file with a New-Line.